src/ui/primitives/tooltip.rs

```diff
@@ -38,10 +38,6 @@ impl TooltipSignals {
 		self.que.tooltip.update(|item| item.retain(|ids| *ids != id));
 	}
 
-	pub fn unque_all_tooltips(self) {
-		self.que.tooltip.set(Vec::new());
-	}
-
```

Comment:
Felt like this was not the right place for this so I moved it to the Que instance